jpegenc/subblocks/2d_dct/test_1d_dct.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+import myhdl
+from myhdl import (StopSimulation, block, Signal, ResetSignal, intbv,
+                                      delay, instance, always_comb, always_seq)
+from myhdl.conversion import verify
+
+from dct_1d import input_interface, output_interface, dct_1d
+
+test_block =  [0xa6, 0xa1, 0x9b, 0x9a, 0x9b, 0x9c, 0x97, 0x92,
+               0x9f, 0xa3, 0x9d, 0x8e, 0x89, 0x8f, 0x95, 0x94,
+               0xa5, 0x97, 0x96, 0xa1, 0x9e, 0x90, 0x90, 0x9e,
+               0xa7, 0x9b, 0x91, 0x91, 0x92, 0x91, 0x91, 0x94,
+               0xca, 0xda, 0xc8, 0x98, 0x85, 0x98, 0xa2, 0x96,
+               0xf0, 0xf7, 0xfb, 0xe8, 0xbd, 0x96, 0x90, 0x9d,
+               0xe9, 0xe0, 0xf1, 0xff, 0xef, 0xad, 0x8a, 0x90,
+               0xe7, 0xf2, 0xf1, 0xeb, 0xf7, 0xfb, 0xd0, 0x97]
+
+@myhdl.block
```

Comment:
the `clock_driver` and the `reset_on_start` should be put in a separate module and imported in the tests, since it is used in more than one test.  In the test directory there is a local package `support`, you could put them somewhere in the `support` package.

The usage in the test modules would be

```python
from support import clock_driver, reset_on_start
```